scripts/sscm.py

```diff
@@ -0,0 +1,773 @@
+    parser_create_cluster.add_argument("-n", "--nodes", type=int, action="store", default=0,
+            help="amount of nodes to create the cluster with, the nodes are not started")
+    parser_create_cluster.add_argument("--network-prefix", type=int, action="store", default=None,
+            help="ip addresses for nodes will be constructed as 127.0.$network_prefix.X, if not provided, a random number will be used")
+
+    parser_start_cluster = subparsers.add_parser(
+            "start-cluster",
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=textwrap.dedent(f"""\
+            Start an existing cluster
+
+            Starts the supervisor and optionally start all the nodes.
+            Use stop-cluster to stop the supervisor (and the nodes).
+            """),
+            help="start the supervisor for an existing cluster (doesn't start the nodes themselves)")
+
+    cluster_common_args = [
+            (("-a", "--start-all"), {"action": "store_true", "default": None,
+                    "help": "start all nodes added to the cluster, see also --start-all-pacing"})]
+
+    for args, kwargs in cluster_common_args:
+        parser_create_cluster.add_argument(*args, **kwargs)
+        parser_start_cluster.add_argument(*args, **kwargs)
+
+    parser_stop_cluster = subparsers.add_parser(
+            "stop-cluster",
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=textwrap.dedent(f"""\
+            Stop the cluster
+
+            Stops all the nodes, then stops the supervisor.
+            Use start-cluster to start the supervisor again.
+            """),
+            help="stop all nodes and stop the supervisor")
+
+    parser_add_node = subparsers.add_parser(
+            "add-node",
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=textwrap.dedent(f"""\
+            Add a new node to the cluster
+
+            Creates a new directory for the node, called node{{i}}, where i is the next
+            available index. The assigned index is printed on success.
+            The node's directory is populated with a Scylla binary, a log file for Scylla
+            ({SCYLLA_LOG_FILE}), and any other directories and resources required to run a
+            Scylla node.
+
+            The node is not started, you can do that with start-node.
+            """),
+            help="add a new node to the cluster")
+    parser_add_node.add_argument("-n", "--nodes", type=int, action="store", default=1,
+            help="amount of nodes to add")
+
+    create_add_common_args = [
+        (("-p", "--path"), {"help": "path to the Scylla repository to use"}),
+        (("--mode",), {"default": "dev", "choices": ["coverage", "dev", "debug", "release"], "help": "Scylla build mode to use"}),
+        (("-c", "--smp"), {"type": int, "default": 1, "help": "number of cpus to start Scylla with"}),
```

Comment:
Better default is 2, to exercise smp a little.

Reply:
Done in v2.